examples/stable_diffusion_v2/image2image.py

```diff
@@ -9,10 +9,10 @@
 
 import cv2
 import numpy as np
-from annotator.canny import CannyDetector
-from annotator.util import HWC3, resize_image
 from cldm.ddim_hacked import DDIMSampler
 from cldm.model import create_model, load_model
+from conditions.controlnet import CannyDetector, SegmentDetector
+from conditions.controlnet.utils.utils import HWC3, resize_image
```

Comment:
define `HWC3`, `resize_image` inside `utils.__init__.py` and you can avoid `from utils.utils`.